test/test_list.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# set -uo pipefail;
```

Comment:
I see you commented this, but the `-o pipefail` is actually useful to catch code errors. The `-u` is not actually very useful because we have empty variables. `-o pipefail` could actually be used in every test I believe.

Also, instead of comment code if we don't want we should just remove.

Reply:
Ack - sorry I am normally better at going back and cleaning this up. I've uncommented pipefail and deleted other commented code.